deployments/neo4j.yml

```diff
@@ -0,0 +1,92 @@
+---
+  apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: nitro-neo4j
+    namespace: staging
+  spec:
+    replicas: 1
+    strategy: {}
+    selector:
+      matchLabels:
+        workload.user.cattle.io/workloadselector: deployment-staging-neo4j
+    template:
+      metadata:
+        labels:
+          workload.user.cattle.io/workloadselector: deployment-staging-neo4j
+        name: nitro-neo4j
```

Comment:
This is the name that is used for connecting to the db internally via the local network `bolt://nitro-neo4j:7687`

Reply:
Exactly, that was the problem. :wink: :+1: